src/static/translations/customer-portal/fr.json

```diff
@@ -0,0 +1,223 @@
+    "address_plural": "Adresses",
+    "cancel": "Annuler",
+    "close": "Fermer",
+    "payment_method_plural": "Méthodes de paiement",
+    "save": "Sauvegarder",
+    "subscription_plural": "Abonnements",
+    "transaction_plural": "Transactions",
+    "undo_cancel": "Revoir",
+    "undo_confirm": "Abandon",
+    "undo_header": "Modifications non sauvegardées",
+    "undo_message": "On dirait que vous n'avez pas enregistré vos modifications ! Que voulez-vous en faire ?",
+    "update": "Mettre à jour",
+    "address-card": {
+      "default_billing_address": "Adresse de facturation par défaut",
+      "default_shipping_address": "Adresse de livraison par défaut",
+      "full_address": "{{address1}} {{address2}} {{postal_code}} {{city}} {{region}}",
+      "full_name": "{{first_name}} {{last_name}}",
+      "spinner": {
+        "loading_busy": "Chargement",
+        "loading_empty": "Aucune adresse",
+        "loading_end": "Plus aucune donnée",
+        "loading_error": "Échec du chargement"
+      }
+    },
+    "address-form": {
+      "address_name": "Nom de l'adresse",
+      "address1": "Ligne d'adresse 1",
+      "address2": "Ligne d'adresse 2",
+      "cancel": "Annuler",
+      "city": "Ville",
+      "company": "Société",
+      "country": "Pays",
+      "date": "{{value, date}}",
+      "date_created": "Créé le",
+      "date_modified": "Dernière mise à jour",
+      "delete": "Supprimer",
+      "delete_prompt": "Cette ressource sera définitivement supprimée. Vous êtes sûr ?",
+      "first_name": "Prénom",
+      "last_name": "Nom",
+      "phone": "Téléphone",
+      "postal_code": "Code postal",
+      "region": "Région",
+      "v8n_required": "Requis",
+      "v8n_too_long": "Trop long",
+      "spinner": {
+        "loading_busy": "Chargement",
+        "loading_empty": "Pas de données",
+        "loading_error": "Échec du chargement"
+      }
+    },
+    "customer-form": {
+      "cancel": "Annuler",
+      "change_password": "Modifier le mot de passe",
+      "change_password_step_1": "Commençons par votre mot de passe actuel. Un mot de passe temporaire fonctionne également si vous en avez un.",
+      "change_password_step_2": "Saisissez maintenant un nouveau mot de passe et cliquez sur Continuer pour appliquer les changements à votre compte.",
+      "change_password_step_3": "C'est fait ! Vous pouvez maintenant utiliser votre nouveau mot de passe pour vous connecter.",
+      "close": "Fermer",
+      "date": "{{value, date}}",
+      "date_created": "Créé le",
+      "date_modified": "Dernière mise à jour",
+      "email": "E-mail",
+      "first_name": "Prénom",
+      "invalid_credential_error": "Le mot de passe est incorrect. Vérifiez que vous avez bien saisi votre mot de passe actuel ou temporaire et réessayez.",
+      "last_name": "Nom",
+      "new_password": "Nouveau mot de passe",
+      "password": "Mot de passe",
+      "sign_in": "Continuer",
+      "tax_id": "Siren",
+      "undo_cancel": "REvoir",
```

Comment:
@rijarobinson This might be a small typo?

Reply:
Fixed